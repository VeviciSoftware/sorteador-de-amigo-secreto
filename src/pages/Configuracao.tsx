import Formulario from "../components/Formulario/Formulario";
import ListaParticipantes from "../components/ListaParticipantes/ListaParticipante";
import Rodape from "../components/Rodape/Rodape";

const Configuracao = () => {
    return (
        <>
            <Formulario />
            <ListaParticipantes />
            <Rodape />
        </>
    );
}

export default Configuracao;