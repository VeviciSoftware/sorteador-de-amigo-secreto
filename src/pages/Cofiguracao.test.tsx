import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Configuracao from "./Configuracao";

describe("a pagina de configuração", () => {
  test("deve ser renderizada corretamente", () => {
    const { container } = render(
      <RecoilRoot>
        <Configuracao />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();


  });
});
