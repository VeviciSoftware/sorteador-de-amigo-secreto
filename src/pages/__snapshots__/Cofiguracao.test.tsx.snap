// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuração deve ser renderizada corretamente 1`] = `
<div>
  <confi />
</div>
`;
